config/params/config.go

```diff
@@ -763,3 +774,23 @@ func WithinDAPeriod(block, current primitives.Epoch) bool {
 
 	return block+BeaconConfig().MinEpochsForBlobsSidecarsRequest >= current
 }
+
+// EpochsDuration returns the time duration of the given number of epochs.
+func EpochsDuration(epochs primitives.Epoch, b *BeaconChainConfig) time.Duration {
```

Comment:
It feels strange to have a variable `epochs` (wish terminal `s`) which is of typ `primitives.Epoch`.
I think for clarity `epochCount` would really indicate we do not expect and epoch (number) here, but a number of epoch.